__tests__/shapes.test.js

```diff
@@ -0,0 +1,60 @@
+const PPTX = require('../index.js');
+const fs = require('fs');
+//const path = require('path');
```

Comment:
can we delete this if we don't need it?

Reply:
It's for the dir cleaning function which is temporarily commented out. Sometimes I run that test on it's own.

Reply:
Nevermind. I deleted the "scrubber" functions anyway. It's easier this way.